src/Sylius/Behat/Page/Checkout/CheckoutSecurityStep.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page\Checkout;
+
+use Sylius\Behat\SymfonyPageObjectExtension\PageObject\SymfonyPage;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class CheckoutSecurityStep extends SymfonyPage
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getRouteName()
+    {
+        return 'sylius_checkout_security';
+    }
+
+    /**
+     * @param string $login
+     * @param string $password
+     */
+    public function logInExistingUser($login, $password)
```

Comment:
Ok, that seems a little bit more complicated than I thought :D Anyway, we can rename it to `logInAsExistingUser` to be more semantically correct.